atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -96,6 +96,8 @@ def __init__(self, tasker, workflow, kojihub, url, build_json_dir,
         :param koji_principal: str, Kerberos principal (must specify keytab)
         :param koji_keytab: str, keytab name (must specify principal)
         :param blocksize: int, blocksize to use for uploading files
+        :param prefer_schema1_digest: bool, when True, schema v1 digest is
+            will be preferred as the built image digest
```

Comment:
Little grammar nitpick: "is will be preferred"

Reply:
Thanks! fixed.